### PR TITLE
Support UICollectionViews embedded in UITableViewCells

### PR DESCRIPTION
**Pull request by alliannas**
Updates Podfiles to 1.0.0+ syntax

Adds supports for caching the `contentOffset` properties of UICollectionView inside UITableViewCells.
Otherwise the contentOffset is reused which causes confusion for users.

### CODE REVIEW

File: .idea/vcs.xml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="VcsDirectoryMappings">
+    <mapping directory="$PROJECT_DIR$" vcs="Git" />
+  </component>
+</project>
```
Comment (johannth) [inline .idea/vcs.xml]: I'm not sure these .idea files should be part of this repo.

File: PVGTableViewProxy/PVGTableViewProxy.m [hunk 2]
```diff
@@ -321,6 +337,19 @@ - (void)tableView:(UITableView *)tableView willDisplayCell:(UITableViewCell *)ce
...
     
+    if ([cell conformsToProtocol:@protocol(PVGTableViewCellWithCollectionView)])
+    {
+        id<PVGTableViewCellWithCollectionView> cellWithCollectionView = (id<PVGTableViewCellWithCollectionView>)cell;
+        UICollectionView *collectionView = cellWithCollectionView.collectionView;
+        
+        id<PVGTableViewCellViewModel> cellViewModel = [self viewModelForIndexPath:indexPath];
+        
+        NSString *cacheID = cellViewModel.cacheID;
+        CGFloat horizontalOffset = self.collectionViewContentOffset[cacheID].floatValue;
```
Comment (johannth) [inline PVGTableViewProxy/PVGTableViewProxy.m]: This is very specific feature. What about the vertical offset, should we cache that as well?

**Comment by johannth** [comment]
Looks good. I think we should add either tests or an example that reproduces this bug before merging this.

**Comment by alliannas** [comment]
@johannth I've added support for caching the whole content offset + added tests

**Pull request merged**